.github/workflows/continuous-integration.yml

```diff
@@ -9,3 +13,20 @@ on:
 jobs:
   ci:
     uses: laminas/workflow-continuous-integration/.github/workflows/continuous-integration.yml@1.x
+
+  composer-require-checker:
+    name: "Check for missing dependencies"
```

Comment:
Wait, we don't have this in our CI tooling? :thinking:

Reply:
Yes - it's available, but deps are always conflicting - Doing it separately means I can run it on 8.2 (at the latest version), and I don't get blocked upgrading Psalm/PHPUnit to latest possible with PHP constraints